examples/addAsset.http

```diff
@@ -0,0 +1,10 @@
+POST http://localhost:8080/users/1/favourites
```

Comment:
You're welcome. I haven't used Thunder Client, but as I see it, it can truly consume them. I have provided the simplest form of a request in order to enable you use them with any client you want.